packages/mobile/src/screens/audio-screen/AudioScreen.tsx

```diff
@@ -224,23 +224,23 @@ export const AudioScreen = () => {
     )
   }
 
-  const onPressSend = useCallback(() => {
-    dispatchWeb(
+  const handlePressSend = useCallback(() => {
```

Comment:
just curious, should we standardize on `handle` instead of `on`?

Reply:
imo yeah! it's fairly minor nit i have tho

Reply:
sounds good to me. client sync that ish